debian/appscale_install_functions.sh

```diff
@@ -15,12 +15,17 @@ if [ -z "${APPSCALE_PACKAGE_MIRROR-}" ]; then
     export APPSCALE_PACKAGE_MIRROR=http://s3.amazonaws.com/appscale-build
 fi
 
+JAVA_VERSION="java-8-openjdk"
```

Comment:
OpenJDK 7 is not available in Xenial's repos, but the Java 8 runtime will run Java 7 apps. I haven't tested if apps that take advantage of Java 8 features work.